### ISSUE TITLE
SQL ingestion fails when Instance name contains single quote

### ISSUE THREAD

**Issue by UserY03** [helderpinto/AzureOptimizationEngine#127]
see issue #21

**Comment by helderpinto** [comment]
Is this a regression of the same bug? Can you provide more details, like runbook errors or a (masked) JSON recommendations file generated in your environment that is causing this failure?

**Comment by UserY03** [comment]
So I just added a new rule like the solution of the previous issue, only changing so that is works for the Instance name 
if ($jsonObjectSplitted[$j][$i].InstanceName -ne $null) {
                       $jsonObjectSplitted[$j][$i].InstanceName = $jsonObjectSplitted[$j][$i].InstanceName.Replace("'", "")
                         } 
*note: it also gave an error if there was not a Instance name so therefor I used a if statement

**Comment by UserY03** [comment]
this is the error I got before implementing my solution: 
Exception calling "ExecuteReader" with "0" argument(s): "Incorrect syntax near 's'. The floating point value '2e8562228' is out of the range of computer representation (8 bytes). Incorrect syntax near the keyword 'with'. If this statement is a common table expression, an xmlnamespaces clause or a change tracking context clause, the previous statement must be terminated with a semicolon. The label 'https' has already been declared. Label names must be unique within a query batch or stored procedure. The label 'T06' has already been declared. Label names must be unique within a query batch or stored procedure. Incorrect syntax near the keyword 'with'. If this statement is a common table expression, an xmlnamespaces clause or a change tracking context clause, the previous statement must be terminated with a semicolon.

**Comment by helderpinto** [comment]
Thanks, @UserY03 . Can you give an example of such an instance name/recommendation, so that I am able to reproduce the error?

**Comment by UserY03** [comment]
In our company the problem creator was named: let's encrypt

**Comment by helderpinto** [comment]
Thanks for reporting this! This issue will be fixed soon and will be closed automatically once the code is merged into the main branch.

**Comment by UserY03** [comment]
alright, thanks for the fast responses 
have a great day
